net-p2p/eiskaltdcpp/metadata.xml

```diff
@@ -5,10 +5,6 @@
 		<email>klondike@gentoo.org</email>
 		<name>Francisco Blas (klondike) Izquierdo Riera</name>
 	</maintainer>
-	<maintainer type="project">
-		<email>proxy-maint@gentoo.org</email>
-		<name>Proxy Maintainers</name>
-	</maintainer>
```

Comment:
Just leave it for now, let's try to solve it with pkgcheck upstream.

Reply:
Readded this :)